datafusion/functions/src/string/mod.rs

```diff
@@ -57,11 +61,21 @@ pub mod expr_fn {
         super::ascii().call(vec![arg1])
     }
 
+    #[doc = "Returns the number of bits in the `string`"]
+    pub fn bit_length(args: Vec<Expr>) -> Expr {
```

Comment:
See above -- I think this should take a single `Expr` argument rather than a `Vec<Expr>`